app/design/adminhtml/Magento/backend/web/css/styles-old.less

```
@@ -3962,7 +3962,7 @@
 }
 
 //  Configuration -> Payment Methods
-.adminhtml-system-config-edit {
+.adminhtml-system_config-edit {
```

Let's keep existing selector for backward compatibility
```suggestion
.adminhtml-system-config-edit,
.adminhtml-system_config-edit {
```

---

thanks, changed it.